### PR TITLE
Improve error log when setting pricing conf via extra data fails

### PR DESCRIPTION
**Pull request by fab-10**

### CODE REVIEW

File: sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java [hunk 1]
```diff
@@ -75,10 +75,11 @@ public void start() {
         extraDataHandler.handle(initialExtraData);
       } catch (final Exception e) {
         log.warn(
-            "Failed setting initial pricing conf from extra data field ({}) of the chain head block {}({})",
+            "Failed setting initial pricing conf from extra data field ({}) of the chain head block {}({}), error {}",
```
Comment (Filter94) [inline sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java:78]: Can we log the whole stacktrace on a lower log level?

Comment (fab-10) [reply sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java:78]: is it ok also if the stack trace is logged in this warn statement, without adding a new one?

Comment (Filter94) [reply sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java:78]: I'm not sure about that TBH, up to you. But in the vast majority of cases just the error message should be sufficient.

Comment (fab-10) [reply sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java]: the new version logs the error message and the stack trace

**Review by Filter94 — APPROVED** [review]

**Pull request merged**